app/views/users/show.html.erb

```diff
@@ -7,3 +8,36 @@
     </ul>
   </article>
 </section>
+
+<div class="d-flex">
+  <section class="user-section mx-3">
+  <h2>friends</h2>
+  <ul class="posts">
+  <%@user.reload.friends.each do |friendship|%>
+  <li><%=friendship.name.capitalize%></li>
+  <%end%>
+  </ul>
+
+  </section>
+  <section class="user-section text-nowrap">
+    <h2>Pending friends</h2>
+    <ul class="posts">
+      <%@user.pending_friends.each do |friend|%>
```

Comment:
**[REQUIRED]**
* In order to accept a friend request, we need to find the friend_requests of the `current_user`
We can change this to 
```ruby
current_user.friend_requests
```